api/app.py

```diff
@@ -1,20 +1,64 @@
-from fastapi import FastAPI, HTTPException, status, File, UploadFile, Form
+from fastapi import FastAPI, HTTPException, status, Depends, File, UploadFile, Form
+from fastapi.middleware.trustedhost import TrustedHostMiddleware
 from fastapi.openapi.utils import get_openapi
+from fastapi.security import OAuth2PasswordBearer
 from fastapi_health import health
+import jwt
+
+import json
+
+from duckdb import DuckDBPyConnection
 
 import substrait_validator as sv
 
 from backend.duckdb import (
-    ConnectDuckDB,
+    DuckDBConnection,
     CheckDuckDBConnection,
     ExecuteDuckDb,
     ParseFromDuckDB,
+    DeleteTableFromDuckDB,
 )
 
+from backend.ttl_cache import TTL_Cache
+
 from loguru import logger
 
+
 app = FastAPI()
-con = None
+
+allowed_urls = [
+    "substrait-fiddle.com",
+    "substrait-fiddle.com",
```

Comment:
This is the same as the line before, isn't it?

Reply:
yep, sorry, repetition error, fixed that.